includes/class-wc-payments-account.php

```diff
@@ -394,7 +394,15 @@ public static function get_login_url() {
 	 * @return string Stripe account login url.
 	 */
 	public static function get_connect_url() {
-		return wp_nonce_url( add_query_arg( [ 'wcpay-connect' => '1' ] ), 'wcpay-connect' );
+		$connect_url = add_query_arg(
+			[
+				'page' => 'wc-admin',
+				'path' => '/payments/connect',
```

Comment:
Is this a route added somewhere or just a placeholder?  Some updates to returned connections here - https://github.com/woocommerce/woocommerce-admin/pull/7095

Reply:
This directs to the primary connect page, but the `wcpay-connect` parameter then redirects to the onboarding flow with Stripe.